lib/BackbeatProducer.js

```diff
@@ -54,7 +54,7 @@ class BackbeatProducer extends EventEmitter {
         const { zookeeper, sslOptions, topic, partition } = validConfig;
 
         this._partition = partition;
-        this._zookeeperEndpoint = zookeeper.connectionString;
+        this._zookeeperEndpoint = `${zookeeper.connectionString}/backbeat`;
```

Comment:
Why are we changing this line?

Reply:
I had changed it for testing locally, but reverted it with the latest commit for CI tests. So it remains unchanged in the target branch.